### PR TITLE
Add last version of RobotDynamicsEstimator code used for the paper - PR1

### PR DESCRIPTION
**Pull request by isorrentino**
Hi @GiulioRomualdi @S-Dafarra, 
this PR wants to align the files of the `RobotDynamicsEstimator` library already merged in `master` with the last version used for the paper. The changes in this PR are the consequence of the code optimization.

### CODE REVIEW

File: src/Estimators/include/BipedalLocomotion/RobotDynamicsEstimator/AccelerometerMeasurementDynamics.h [hunk 1]
```diff
@@ -144,4 +148,4 @@ BLF_REGISTER_UKF_DYNAMICS(AccelerometerMeasurementDynamics,
 } // namespace Estimators
 } // namespace BipedalLocomotion
 
-#endif // BIPEDAL_LOCOMOTION_ESTIMATORS_ACCELEROMETER_MEASUREMENT_DYNAMICS_H
+#endif // BIPEDAL_LOCOMOTION_ESTIMATORS_ACCELEROMETER_MEASUREMENT_DYNAMICS_H
```
Comment (GiulioRomualdi) [inline src/Estimators/include/BipedalLocomotion/RobotDynamicsEstimator/AccelerometerMeasurementDynamics.h:151]:
```suggestion
#endif // BIPEDAL_LOCOMOTION_ESTIMATORS_ACCELEROMETER_MEASUREMENT_DYNAMICS_H

```

File: src/Estimators/include/BipedalLocomotion/RobotDynamicsEstimator/ConstantMeasurementModel.h [hunk 2]
```diff
@@ -104,4 +104,4 @@ BLF_REGISTER_UKF_DYNAMICS(ConstantMeasurementModel, ::BipedalLocomotion::Estimat
 } // namespace Estimators
 } // namespace BipedalLocomotion
 
-#endif // BIPEDAL_LOCOMOTION_ESTIMATORS_CONSTANT_MEASUREMENT_MODEL_H
+#endif // BIPEDAL_LOCOMOTION_ESTIMATORS_CONSTANT_MEASUREMENT_MODEL_H
```
Comment (GiulioRomualdi) [inline src/Estimators/include/BipedalLocomotion/RobotDynamicsEstimator/ConstantMeasurementModel.h:107]: Can you add the a white line at the end of the file?

**Review by GiulioRomualdi — CHANGES_REQUESTED** [review]

**Comment by isorrentino** [comment]
@GiulioRomualdi I added the new line at the end of each file.

**Comment by GiulioRomualdi** [comment]
Thank you @isorrentino. Let's wait for the CI

**Review by GiulioRomualdi — APPROVED** [review]

**Pull request merged**